src/test/java/core/basesyntax/service/strategy/strategyimpl/BalanceOperationTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private BalanceOperation balanceOperation;
+    private FruitRecord fruitRecord;
+
+    @BeforeEach
+    public void setUp() {
+        balanceOperation = new BalanceOperation();
+        Storage.storage.clear();
+    }
+
+    @Test
+    void testApply_ShouldAddFruitToStorage() {
```

Comment:
```suggestion
    void testApply_shouldAddFruitToStorage_success() {
```
use lowercase after underscore, stick to the naming convention - name should have 3 parts, third is about expected outcome